inform-annotations/pom.xml

```
@@ -0,0 +1,90 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>inform-annotations</artifactId>
+	<version>1.0.0</version>
+	<name>Inform Annotations</name>
+	<packaging>jar</packaging>
+	<description>Inform-DB Spring JPA Annotation Helpers</description>
+
+	<parent>
+		<groupId>uk.ac.ucl.rits.inform</groupId>
+		<artifactId>emap-star</artifactId>
+		<version>1.0</version>
+	</parent>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<maven.compiler.target>11</maven.compiler.target>
+		<maven.compiler.source>11</maven.compiler.source>
+		<auto-service.version>1.0-rc2</auto-service.version>
+		<maven-compiler-plugin.version> 3.5.1</maven-compiler-plugin.version>
+		<hibernate.core.version>5.4.21.Final</hibernate.core.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>com.google.auto.service</groupId>
+			<artifactId>auto-service</artifactId>
+			<version>${auto-service.version}</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.hibernate</groupId>
+			<artifactId>hibernate-core</artifactId>
+			<version>${hibernate.core.version}</version>
+		</dependency>
+	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<!-- Needed for annotation processing -->
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>${maven-compiler-plugin.version}</version>
+				<configuration>
+					<source>${maven.compiler.source}</source>
+					<target>${maven.compiler.target}</target>
+				</configuration>
```

Still need this for intelliJ to play nicely with this
```suggestion
				</configuration>
								<executions>
					<execution>
						<id>default-compile</id>
						<configuration>
							<compilerArgument>-proc:none</compilerArgument>
							<includes>
								<include>uk/ac/ucl/rits/inform/informdb/annotation/AuditTableProcessor.java</include>
							</includes>
						</configuration>
					</execution>
					<execution>
						<id>compile-project</id>
						<phase>compile</phase>
						<goals>
							<goal>compile</goal>
						</goals>
					</execution>
				</executions>
```